### PR TITLE
Tom fix routing

### PR DESCRIPTION
**Pull request by tladendo**
fyi @indytechcook @vietnogi – this turns the getting-started module into a proper Angular module and turns the shell into a shell for ui-router. I still need to make it so that we can dynamically inject the mock stuff, but this took me longer to figure out than I thought it would.

### CODE REVIEW

File: src/client/app/getting-started/getting-started.controller.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*global form:true */
+(function () {
+  'use strict';
```
Comment (vietnogi) [inline src/client/app/getting-started/getting-started.controller.js]: Can we organize js files in a js/script folder.

File: src/client/app/layout/views/main.jade [hunk 2]
```diff
@@ -1,2 +1,2 @@
 main.layout-main("role"="main", class="getting-started-page")
-  include ../../getting-started/getting-started.jade
+  div(ui-view='')
```
Comment (vietnogi) [inline src/client/app/layout/views/main.jade]: double quotes please

**Comment by vietnogi** [comment]
Minor comments, otherwise, lgtm.

**Comment by indytechcook** [comment]
@tladendo does this setup routing using # or #!?  We need those for hosting to work from S3

**Comment by tladendo** [comment]
I'll fix this stuff up tomorrow morning – thanks for the comments.

**Comment by indytechcook** [comment]
otherwise lgtm

**Pull request merged**